Bottom: 69235e292c40952c74daee25b5080ceea029b2ce
Top:    d0269fd6397d291b2f59fd0c779bb2e8856b1604
Author: Peter Clifton <peter@clifton-electronics.co.uk>
Date:   2016-12-04 20:31:10 +0000

DEBUG: Polygon parentage on


---

diff --git a/src/polygon1.c b/src/polygon1.c
index 8041c05..efece45 100644
--- a/src/polygon1.c
+++ b/src/polygon1.c
@@ -2964,13 +2964,14 @@ poly_Boolean_free (POLYAREA * ai, POLYAREA * bi, POLYAREA ** res, int action)
 
   *res = NULL;
 
-#if 0
+#if 1
   /* Make copies for tracking polygon parentage (DEBUG) */
   if (!poly_M_Copy0 (&a_copy, a) || !poly_M_Copy0 (&b_copy, b))
       return err_no_memory;
-#endif
+#else
   a_copy = NULL;
   b_copy = NULL;
+#endif
 
   /* Move the parentage information over onto the copy */
   if (a_copy != NULL)
